Makefile

```diff
@@ -0,0 +1,45 @@
+DIST_VERSION ?= 7
+
+NAME := leapp-supplements
+VERSION := 1.0.0
```

Comment:
why? The version is defined now 2x - once here and once in the specfile. Make your life easier and do not do that :)

Reply:
Good point, used the grep command from leapp-repository :) `grep -m1 "^Version:" packaging/$(PKGNAME).spec | grep -om1 "[0-9].[0-9.]**"`